tests/multi-region/multi_region.test.js

```diff
@@ -1,71 +1,82 @@
 'use strict';
 
-const path = require('path');
 const fs = require('fs');
-const axios = require('axios');
+const path = require('path');
+
 const { expect } = require('chai');
+const { describe, it } = require('@jest/globals');
+
 const { execSync } = require('../../shared/child-process');
 const { getTmpDirPath, replaceTextInFile } = require('../utils/fs');
-const { getServiceName, sleep, serverlessDeploy, serverlessRemove} = require('../utils/misc');
-const { FunctionApi, RegistryApi } = require('../../shared/api');
-const { REGISTRY_API_URL, FUNCTIONS_API_URL } = require('../../shared/constants');
+const { getServiceName, serverlessDeploy, serverlessRemove, serverlessInvoke, createProject, sleep } = require('../utils/misc');
+const { FunctionApi } = require('../../shared/api');
+const { FUNCTIONS_API_URL } = require('../../shared/constants');
+const { removeProjectById } = require('../utils/clean-up');
 
 const serverlessExec = path.join('serverless');
 
+const scwToken = process.env.SCW_SECRET_KEY;
+
 const functionTemplateName = path.resolve(__dirname, '..', '..', 'examples', 'python3');
 const oldCwd = process.cwd();
 const serviceName = getServiceName();
-const scwProject = process.env.SCW_DEFAULT_PROJECT_ID || process.env.SCW_PROJECT;
-const scwToken = process.env.SCW_SECRET_KEY || process.env.SCW_TOKEN;
-let apiUrl;
-let api;
-let namespace;
 
 const regions = ['fr-par', 'nl-ams', 'pl-waw'];
 
-describe.each(regions)(
-  'test regions',
-  (region) => {
-    it(`should deploy service for region ${region}`, async () => {
-      const tmpDir = getTmpDirPath();
+describe("test regions", () => {
 
-      // create working directory
+  it.concurrent.each(regions)(
+  'region %s',
+  async (region) => {
+
+      let options = {};
+      options.env = {};
+      options.env.SCW_SECRET_KEY = scwToken;
+
+      let projectId;
+      let apiUrl;
+      let api;
+      let namespace;
```

Comment:
you can also do `let projectId, apiUrl, ...` but it doesn't really matter

However, I wonder if this codebase is linted? Now that we maintain more Javascript projects, we can try to have the same linter config between them.